rest-helix/src/main/java/com/github/twitch4j/helix/domain/Emote.java

```diff
@@ -211,4 +233,55 @@ public static Type parseEmoteType(String emoteType) {
 
     }
 
+    public enum Format {
+
+        /**
+         * Returns an animated GIF if available, otherwise, returns the static PNG.
+         */
+        @JsonEnumDefaultValue
+        DEFAULT,
+
+        /**
+         * Indicates a static PNG file is available for this emote.
+         */
+        STATIC,
+
+        /**
+         * Indicates an animated GIF is available for this emote.
+         */
+        ANIMATED
+
+    }
+
+    @RequiredArgsConstructor
+    public enum Scale {
+
+        /**
+         * A small version (28px x 28px) of the emote.
+         */
+        @JsonProperty("1.0")
+        SMALL("1.0"),
+
+        /**
+         * A medium version (56px x 56px) of the emote.
+         */
+        @JsonProperty("2.0")
+        MEDIUM("2.0"),
+
+        /**
+         * A large version (112px x 112px) of the emote.
+         */
+        @JsonProperty("2.0")
+        LARGE("2.0");
```

Comment:
I think this should be `3.0` for large.

Reply:
you're correct, I shouldn't be coding at 2am lol